### ISSUE TITLE
--eval breaks on using a Python reserved keyword as a parameter

### ISSUE THREAD

**Issue by ravikishore1993** [sqlmapproject/sqlmap#1062]
Sqlmap breaks when the testing paramter names are Python reserved keywords and `eval` is being used.

Exact sqlmap query:
```
python sqlmap.py -u http://MyWebsite.org/ \
           --data="token=e44f81087134931a&pass=lslsl" -p pass\
           --eval="import requests;a=requests.get('http://MyWebsite.org/');b=str(a.text);token=b[352:368];print token;"
```

The paramters passed in POST request are `token` and `pass`.    
`pass` is a [reserved](https://docs.python.org/2.5/ref/keywords.html) Python keyword. 
 `eval` is used here to modify `token` parameter on the fly, since it varies per request.
The ouput :- 

```
➜  sqlmap  python sqlmap.py -u http://otp.adctf2014.katsudon.org/ \
>            --data="token=e44f81087134931a&pass=lslsl" -p pass\
>            --eval="import requests;a=requests.get('http://otp.adctf2014.katsudon.org/');b=str(a.text);token=b[352:368];print token;"
         _
 ___ ___| |_____ ___ ___  {1.0-dev-nongit-20141224}
|_ -| . | |     | .'| . |
|___|_  |_|_|_|_|__,|  _|
      |_|           |_|   http://sqlmap.org

[!] legal disclaimer: Usage of sqlmap for attacking targets without prior mutual consent is illegal. It is the end user's responsibility to obey all applicable local, state and federal laws. Developers assume no liability and are not responsible for any misuse or damage caused by this program

[*] starting at 14:17:59

[14:17:59] [INFO] testing connection to the target URL
[14:17:59] [CRITICAL] an error occurred while evaluating provided code ('invalid syntax (<string>, line 1)'). 

[*] shutting down at 14:17:59

➜  sqlmap  


```

I tried debugging the error, It traces back to [this](https://github.com/sqlmapproject/sqlmap/blob/master/lib/core/common.py#L3611-L3622) and [this](https://github.com/sqlmapproject/sqlmap/blob/master/lib/request/connect.py#L833-L854) parts of code of sqlmap.

Basically in [this](https://github.com/sqlmapproject/sqlmap/blob/master/lib/request/connect.py#L843) line, `exec` is triggered as follows, 
```
exec("pass=u'lslsl'")
```

which is a clear violation of Python reserved keyword rule, and thus triggers the error.
Also `pass` keyword is a very common parameter name in the web. 
This needs a fix.

Please comment and set status as required if this is a known bug. I have searched mailing lists and github issues if this was raised previously and posted here.

Cheers
